app/assets/javascripts/pageflow/editor/views/other_entries_collection_view.js

```diff
@@ -18,10 +18,14 @@ pageflow.OtherEntriesCollectionView = Backbone.Marionette.View.extend({
         selection: this.options.selection
       },
       blankSlateViewConstructor: Backbone.Marionette.ItemView.extend({
-        template: 'templates/other_entries_blank_slate'
+        template: 'templates/other_entries_blank_slate',
+        tagName: 'li',
+        className: 'blank_slate'
       }),
       loadingViewConstructor: Backbone.Marionette.ItemView.extend({
-        template: 'templates/loading'
+        template: 'templates/loading',
+        tagName: 'li',
+        className: 'loading'
```

Comment:
On second thought: let's introduce a separate `pageflow.LoadingSpinnerView` to have a reusable component. The `tagName`, `className` and `template` properties can then live inside that view.